samples/tcp-echo/src/main.go

```diff
@@ -25,6 +25,10 @@ import (
 
 // main serves as the program entry point
 func main() {
+	if len(os.Args) != 3 {
+		fmt.Println("you should run it like: 'main 9000,9001 hello' ")
```

Comment:
Maybe change to `main starting_port,ending_port prefix` to give the user a better idea on what the arguments represent.

Reply:
make sense ,  updated it.